automatic/paint.net/tools/ChocolateyInstall.ps1

```diff
@@ -4,9 +4,9 @@ $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
 
 $packageArgs = @{
   PackageName    = $env:ChocolateyPackageName
-  fileType       = 'exe'
-  file64         = Get-Item $toolsPath\*.exe
-  silentArgs     = '/auto'
+  fileType       = 'msi'
+  file64         = Get-Item $toolsPath\*.msi
```

Comment:
We are trying to move away from using globbing patterns of the installer to be used, it would be best to change it to the actual name of the installer, and have the `update.ps1` script automatically replace the name during update.

As such, please update this file to not use a globbing pattern.

See the `calibre` as an example on how to do this:
- `update.ps1`: https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/calibre/update.ps1#L22-L24
- `chocolateyInstall.ps1`: https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/calibre/tools/chocolateyInstall.ps1#L8

Reply:
Done in latest commit.